bsoid_app/predict.py

```diff
@@ -34,7 +31,7 @@ def __init__(self, ROOT_PATH, DATA_DIRECTORIES, input_filenames, processed_input
         self.use_train = []
         self.new_root_path = []
         self.new_directories = []
-        self.filetype = []
+        self.filetype = FTYPE
```

Comment:
`FTYPE` was an array but here is a string. Will this cause issues? Or do you need smth like `self.filetype = [FTYPE]`?

Reply:
think fine as string, originally it is a dropdown selection, but there is only one ftype